Story/StoryHooks.cs

```diff
@@ -572,6 +534,33 @@ private void KarmaLadderScreen_Singal(On.Menu.KarmaLadderScreen.orig_Singal orig
                 orig(self, sender, message);
             }
 
+            private void Player_Update1(ILContext il)
```

Comment:
this hook skips GameOver if the player isn't ours

Reply:
Gotcha. What I'm assuming though is that gameover code should only run for the player dying anyways, so wouldn't this be redundant? No other players are being made aware of current gameover state since that RPC was removed.

Reply:
the issue is that online players can get chomped clientside and trigger GameOver through Player.Update

Reply:
lgtm